src/renderer/globals.scss

```diff
@@ -190,3 +234,33 @@ a.is-button {
 .modal-card {
   z-index: 200;
 }
+
+.button .is-rounded .is-primary {
+  color: $pitch !important;
+  font-weight: 500 !important;
+}
+
+.button.is-primary {
+  color: $pitch !important;
+  font-weight: 500 !important;
+}
+
+.button.is-primary[disabled] {
+  color: $pitch !important;
+  font-weight: 500 !important;
+}
+
+.button.is-rounded.is-cancel {
+  color: $brand-primary !important;
```

Comment:
Where do we use this button? It seems a bit weird to use the green color for the action that sounds risky like cancel

Reply:
this is the text color, not the background
<img width="246" alt="Untitled" src="https://github.com/rfcx/arbimon-uploader/assets/31901584/b49f4ac6-2a79-4daf-adbc-8ed43bbf47e3">

Reply:
<img width="119" alt="Untitled" src="https://github.com/rfcx/arbimon-uploader/assets/31901584/a8178fd4-1345-49f1-858c-8c11f2ef3c06">